crates/biome_lsp/src/handlers/text_document.rs

```diff
@@ -164,16 +164,18 @@ pub(crate) async fn did_change(
         project_key: doc.project_key,
         path: path.clone(),
     })?;
-    debug!("old document: {:?}", old_text);
-    debug!("content changes: {:?}", params.content_changes);
+    // Careful, this logs the entire text document (twice) on almost every keystroke.
+    // Default log level = max debug.
+    trace!("old document: {:?}", old_text);
+    trace!("content changes: {:?}", params.content_changes);
```

Comment:
just making sure, do trace logs make it to the log files by default?

Reply:
Yes for cfg(debug_assertions), no for release builds. See LoggingFilter / SELF_FILTER in daemon.rs.

Reply:
> just making sure, do trace logs make it to the log files by default?

**At the moment**, no they don't because there's not way to set up `trace` as logging level for the daemon

Reply:
I think I'll just delete em. It's only useful if you're implementing the didChange diff applicator for the first time, and that already works.